client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -499,7 +489,7 @@ class EmailProvidersComparison extends Component {
 			translate,
 		} = this.props;
 
-		const formattedPrice = translate( '{{price/}} /user /month billed monthly', {
+		const formattedPrice = translate( '{{price/}} /mailbox /month billed monthly', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.